wiki/markdown/docker.md

```diff
@@ -33,15 +33,15 @@ Now docker image has build with tag "eiffel-intelligence-frontend:0.1"
 To run the produced docker image on the local Docker host, execute this command: 
 
 
-`docker run -p 8034:8091 --expose 8091 -e server.port=8091 -e logging.level.root=DEBUG -e logging.level.org.springframework.web=DEBUG -e logging.level.com.ericsson.ei=DEBUG eiffel-intelligence-backend:0.1`
+`docker run -p 8071:8080 --expose 8080 -e server.port=8080 -e logging.level.log.level.root=DEBUG -e logging.level.org.springframework.web=DEBUG -e logging.level.com.ericsson.ei=DEBUG eiffel-intelligence-frontend:0.0.19`
```

Comment:
Using this command the eiffel intelligence frontend won't know which backend to use, because no backend is specified. Add a flag which passes in the default backend instance to use.

Reply:
I think it should be clearly stated that those values are just examplevalues. Right now it seems like that line should be executed that way, You could otherwise write something like <EXPOSED_PORT>:<CONTAINER_PORT> or something like that for all the "example" values.

Reply:
I agree with @e-edling-ericsson. It would also be easier to maintain the documentation if we have generic example values that don't have to be updated everytime we make a release... 
(thinking of the version tag used in the above command, this PR was written when EI was still 0.0.19 but 1.0.0 has already been released and it will keep changing in the future).